cmake/share/ngraphConfig.cmake.in

```diff
@@ -28,8 +28,9 @@
 
 @PACKAGE_INIT@
 
-include(CMakeFindDependencyMacro)
-find_dependency(Threads)
+if (NOT WIN32)
+    find_package(Threads REQUIRED)
+endif()
```

Comment:
why we cannot use this for WIN32 as well?

Reply:
Because nGraph doesn't use Threads for Windows:
https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/CMakeLists.txt#L734